.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
Should we set this to v18?

Reply:
@jacque006 Yeah I'll put up another PR to bump this up to 18 and remove the requirement to be less than 18 in the extension package.json